test/data/dockerfiles/TestLabels

```diff
@@ -5,7 +5,7 @@
 #
 
 # Pull base image.
-FROM dockerfile/ubuntu
+FROM ubuntu:14.04
```

Comment:
This is not related

Reply:
#### Rule: No tag is used. It is recommended that a specific tag is used in the FROM line or *-released which is the latest supported release.
This test case does not pass this rule

Reply:
Oh, I got it, thanks for the explanation!